optaweb-tsp-ui/src/ducks/tsp/operations.js

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SockJS from 'sockjs-client';
+import webstomp from 'webstomp-client';
+import Creators from './actions';
+
+const {
+  addLocation,
+  deleteLocation,
+  updateTSPSolution,
+  initWsConnection,
+  wsConnectionSuccess,
+  wsConnectionFailure,
+} = Creators;
+
+let webSocket;
+let stompClient;
+
+function mapEventToActions(dispatch) {
```

Comment:
@danielefiungo Is there a good reason to name the function like this? I'm a bit confused. I would expect it to be called perhaps `subscribe()` or `subscribeToRouteTopic()` or similar.